jlibra-core/src/main/java/dev/jlibra/move/Move.java

```diff
@@ -1,11 +1,21 @@
 package dev.jlibra.move;
 
+import com.google.protobuf.ByteString;
+
+import java.io.IOException;
 import java.io.InputStream;
 
 public class Move {
 
-    public static InputStream peerToPeerTransfer() {
-        return Move.class.getResourceAsStream("/move/peer_to_peer_transfer.bin");
-    }
+    public static final ByteString peerToPeerTransfer = peerToPeerTransfer();
```

Comment:
Is there a special reason to have this as public static final instead of having external classes just call the method?

Reply:
As I explained in PR description. This way we don't have to load from file every time you access the function (previously). 
Regardless, now, I think better way would be to insert raw bytes string into a constant directly than loading from file at all. What do you think?

Reply:
Rolled back. Maybe we could resolve this in a separate issue.

Reply:
Didn't see this in the desc, then it's fine, makes sense not reloading on each request (I know it more by assigning on first load to a private field and then checking if this is null on entering the method or fast exiting with the value - but that doesn't matter). I just wouldn't put it into a constant without loading, that's misleading. Loading it once doesn't cost that much time.

Reply:
Well, using static final field is thread safe, while with conventional lazy singleton it is not. Also I'm not sure if reading from the same file concurrently is thread safe, probably not. This may be a bottleneck in the future.

Reply:
Rolled back for now.